server/karaf/features/pom.xml

```diff
@@ -3,8 +3,8 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <groupId>org.apache.james</groupId>
         <artifactId>james-server</artifactId>
+        <groupId>org.apache.james</groupId>
```

Comment:
Not really...

However I notice it only is the case in karaf related project. I'll fix manually, and see if it is stable...